### ISSUE TITLE
CC v5.5 - Accessibilty Issues with the immersive screen reader in new version of MS Teams

### ISSUE THREAD

**Issue by spiritgolfer** [OfficeDev/microsoft-teams-apps-company-communicator#1277]
Hi, 

_A bit of history_: We were using CC V5.4.1 and the change we made basically sent off to the adaptive card library some information so that it could handle the card text in immersive reader. This worked great in the old version of teams.  The immersive screen reader, and JAWS worked as well.

We installed a new instance of CC v5.5 as recommended as the CC app did not work in the new version of MS Teams.
It looks like whatver changes were made, it prevents the immersive screen reader from reading out the speech that we send from the card, using the same fix from the old version.

Any thoughts on how can this be fixed? 
V5.4.1 using the old MS Teams, immersive screen reader works.
V5.5 using the new MS Teams, immersive screen reader does NOT work.

Thanks for your help.

**Comment by gsv022** [comment]
Hi @spiritgolfer ,
Thanks for raising the query. We will check on it and get back to you with proper response soon.

**Comment by gsv022** [comment]
Hi @spiritgolfer ,
Company Communicator complies with Microsoft accessibility standards and guidelines for Web, Windows and Mac OS (Grade C). It has not been tested for immersive reader and supports for only JAWS and NVDA for Web and Desktop versions.

**Comment by spiritgolfer** [comment]
hi @gsv022 , 

Has CC v5.5 been tested with JAWS?  Do you have a record of the testing that you can share?

Thanks!

**Comment by gsv022** [comment]
Hi @spiritgolfer ,
We will get back to you with proper response

**Comment by gsv022** [comment]
Hi @spiritgolfer ,
Thanks for the patience. Unfortunately, we can't provide the recording of the testing. We have tested with latest version of JAWS-J2024.2312.53.400 and it is working.  Please try it out from your side with the specified version and let us know how it goes.

**Comment by spiritgolfer** [comment]
Thank you for your reply.  I have sent along your information and we will be checking our version of JAWS and returning testing results.

**Comment by gsv022** [comment]
@spiritgolfer , yup please keep us posted
